README.md

```diff
@@ -223,6 +234,47 @@ DISTRO_EXTRA_MIDDLEWARES:
   - middleware.test.2
 ```
 
+## How to add extra files requirements
+
+You should set the variable **INSTALL_EXTRA_FILE_REQUIREMENTS** in your config.yml file if you need to install extra files with. The structure should be like:
+
+```yaml
+INSTALL_EXTRA_FILE_REQUIREMENTS:
+  path: ./requirements/extra_file/
+  files: [
+    /edunext/base.txt,
+    /test/test.txt
+  ]
+```
+
+It's important that ``.txt`` files are added in requirements directory, similar to EXTRA PIP REQUIREMENTS from [Tutor](https://docs.tutor.overhang.io/configuration.html#installing-extra-xblocks-and-requirements).
+
+## How to enable openedx extra settings
+
+You should set the variable **OPENEDX_EXTRA_SETTINGS** in your config.yml file if you need to enable ``cms_env``, ``lms_env`` or ``pre_init_lms_task`` settings to plugins works as expected. For now the principals settings should be like this:
+
+```yaml
+OPENEDX_EXTRA_SETTINGS:
+  cms_env: [
+    USE_EOX_TENANT: true
+  ]
+  lms_env: [
+    USE_EOX_TENANT: true,
```

Comment:
I think it will be helpful to add an example of how to add a FEATURE setting because we use allow of that setting.

Reply:
I don't understand your comment here. As explained in the section, this would be the example of how to use the settings. When I mean that the list can grow, it does not depend on the Distro as such, but on what the user needs in the configurations, because the variables are not burned in the code.